trade-finance-manager-api/src/v1/api.js

```diff
@@ -442,6 +442,27 @@ const findFacilitiesByDealId = async (dealId) => {
   }
 };
 
+const getAcknowledgedCompletedAmendments = async (dealId) => {
+  const isValid = isValidMongoId(dealId) && hasValidUri(DTFS_CENTRAL_API_URL);
+  if (isValid) {
+    try {
+      const response = await axios({
+        method: 'get',
+        url: `${DTFS_CENTRAL_API_URL}/v1/tfm/deals/${dealId}/amendments/acknowledged-or-completed`,
```

Comment:
```suggestion
        url: `${DTFS_CENTRAL_API_URL}/v1/tfm/deals/${dealId}/amendments/fully-submitted`,
```

I think we should rename this to allow for future proofing these endpoints and constants.